.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,103 @@
+name: Publish release
```

Comment:
We can simply name it "Publish AsterNixOS". Releasing the ISO and pushing to Cachix serves the ultimate purpose of publishing AsterNixOS. Rename this workflow file accordingly.